src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateAccountCommandTests/WhenICallCreateAccount.cs

```diff
@@ -297,7 +299,13 @@ public async Task ThenAAddedLegalEntityEventIsPublished()
             const string organisationName = "Org";
 
             //Arrange
-            var createAccountCommand = new CreateAccountCommand { PayeReference = "123EDC", AccessToken = "123rd", RefreshToken = "45YT", OrganisationStatus = "active", OrganisationName = organisationName, ExternalUserId = _user.Ref.ToString() };
```

Comment:
Could we use TestCase("Charity", "Charity from Messages Type")
... and the other mappings. Just covers several scenarios

Reply:
Resolved by https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1556/commits/dc01dbe0a97a0c0730c3134f1ae394fffb748e31